### ISSUE TITLE
ggline with numeric x-axis

### ISSUE THREAD

**Issue by mdphan** [kassambara/ggpubr#35]
Hi,

Thanks for this great package. I am trying to use ggline where x-axis is time (time <- c(0,1,3,6,24)) so I need the x-axis values to be treated as numeric, not categorical. However, ggline seems to by default convert x-axis to categorical values. Is there a way to stop it from converting numeric x-axis values to categorical?

Thanks.

**Comment by kassambara** [comment]
New argument `numeric.x.axis` added in `ggline()`. logical. If TRUE, x axis will be treated as numeric. Default is FALSE.

**Comment by SchollJ** [comment]
I'm trying to do something similar: I've got data of say 10 subjects and 5 average measurements of each person, arranged in long format (so columns are ID, measurement bin, measurement value in that bin). I want to plot the data from all subjects together, as well as the average. 

I've tried to plot it with ggerrorplot like this:
ggerrorplot(data, x='measurement_bin', y='measurement_value',add='jitter')
And this works well, the only problem now is that the measurement bins are c(1,2,3,4,10) and they are plotted equally spaced on the x-axis, is there a way to plot them spaced according to their value?
